rocAL/rocAL/source/circular_buffer.cpp

```diff
@@ -142,17 +166,23 @@ void CircularBuffer::pop()
     increment_read_ptr();
     _circ_image_info.pop();
 }
-void CircularBuffer::init(RaliMemType output_mem_type, size_t output_mem_size)
+void CircularBuffer::init(RaliMemType output_mem_type, size_t output_mem_size, size_t buffer_depth)
 {
+    BUFF_DEPTH = buffer_depth;
```

Comment:
please don't use CAPITALS for variables. It is used for const defines

Reply:
Done.